explorer_frontend/src/features/account-connector/init.ts

```diff
@@ -140,7 +140,7 @@ createSmartAccountFx.use(async ({ privateKey, rpcUrl }) => {
         {
           smartAccountAddress: smartAccount.address,
           faucetAddress: tokenFaucetAddress,
-          amount: 10,
+          amount: 10n,
```

Comment:
that's kinda small? if it's wei it's just 0.00...0001

Reply:
hmm, let me check.

Reply:
Because it is not `wei`, it is token without decimal, so it will add exactly 10 tokens.